web-ui/src/pages/BirthdayAnniversaryReportPage.jsx

```diff
@@ -63,22 +63,18 @@ const BirthdayAnniversaryReportPage = () => {
       return;
     }
     if (!birthday) {
-      anniversaryResults = await getAnniversary(months, csrf);
-      setSearchAnniversaryResults(
-        sortAnniversaries(anniversaryResults.payload.data)
-      );
+      anniversaryResults = await getAnniversaries(months, csrf);
+      setSearchAnniversaryResults(sortAnniversaries(anniversaryResults));
       setSearchBirthdayResults([]);
     } else if (!anniversary) {
-      birthdayResults = await getBirthday(months, csrf);
-      setSearchBirthdayResults(sortBirthdays(birthdayResults.payload.data));
+      birthdayResults = await getBirthdays(months, csrf);
+      setSearchBirthdayResults(sortBirthdays(birthdayResults));
```

Comment:
`birthdayResults` is now just an array of objects instead of an array with a `payload` property.